app/validators/placeholders/placeholder_validator.py

```diff
@@ -108,6 +108,19 @@ def validate_option_label_from_value_placeholder(self, answer_id):
                 identifier=answer_id,
             )
 
+    def validate_answer_type_correct(self, argument, transform_type):
+        if argument.get("source") == "answers":
+            transform_type = transform_type.split("_")[1]
```

Comment:
I personally prefer it to be explicit than coupling with our naming scheme, but not a blocker.

Reply:
But how do I do that since we pass two types of transforms into here?